migrate.py

```diff
@@ -282,7 +282,7 @@ def assemble_collections(spec, args):
             'license': None,
             'license_file': None,
             'tags': None,
-            'dependencies': {},
+            'dependencies': None,
             'repository': None,
```

Comment:
Why is this necessary?

Reply:
It isn't. Wanted consistency, but I'll be redoing that code shortly anyway, so going for simple change now. Updated.